packages/caver-kct/src/kip7.d.ts

```diff
@@ -0,0 +1,92 @@
+/*
+    Copyright 2021 The caver-js Authors
+    This file is part of the caver-js library.
+
+    The caver-js library is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Lesser General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    The caver-js library is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Lesser General Public License for more details.
+
+    You should have received a copy of the GNU Lesser General Public License
+    along with the caver-js. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import BigNumber from 'bignumber.js'
+import { Wallet } from '../../..'
+import Contract, { SendData, SendOptions } from '../../caver-contract/src'
+import { ReceiptObject } from '../../caver-rtm/src'
+import { AbiItem } from '../../caver-utils/src'
+
+export interface TokenInfoObject {
+    name: string
+    symbol: string
+    decimals?: number
+    initialSupply?: BigNumber | string
+}
+
+export interface DetectedObject {
+    IKIP7?: boolean
+    IKIP7Metadata?: boolean
+    IKIP7Mintable?: boolean
+    IKIP7Burnable?: boolean
+    IKIP7Pausable?: boolean
+    IKIP17?: boolean
+    IKIP17Metadata?: boolean
+    IKIP17Enumerable?: boolean
+    IKIP17Mintable?: boolean
+    IKIP17MetadataMintable?: boolean
+    IKIP17Burnable?: boolean
+    IKIP17Pausable?: boolean
+    IKIP37: boolean
+    IKIP37Metadata: boolean
+    IKIP37Mintable: boolean
+    IKIP37Burnable: boolean
+    IKIP37Pausable: boolean
+}
+
+export type Amount = string | number | BigNumber
+
+export type Data = string | number | number[] | Buffer
+
+export default class KIP7 extends Contract {
+    constructor(tokenAddress: string, abi?: AbiItem[])
+
+    static byteCode: string
+
+    static deploy(tokenInfo: TokenInfoObject, sendOptions: string | SendOptions): Promise<SendData>
+    static deploy(tokenInfo: TokenInfoObject, sendOptions: string | SendOptions, wallet: Wallet): Promise<SendData> // override from caver-kct
+    static detectInterface(contractAddress: string): Promise<DetectedObject>
```

Comment:
`DetectedObject` should be separated per KCT.

Reply:
Fixed it. please check 0f53ad2